lib/ansible/plugins/action/__init__.py

```diff
@@ -1033,3 +1024,33 @@ def _find_needle(self, dirname, needle):
 
         # if missing it will return a file not found exception
         return self._loader.path_dwim_relative_stack(path_stack, dirname, needle)
+
+    def _get_admin_users(self):
+        '''
+        Returns a list of admin users that are configured for the current shell
+        plugin
+        '''
+        # fallback for old custom plugins w/o get_option
```

Comment:
Move the comment into the except block (since it only apples to admin_users = ['root'], right?)

Reply:
it applies when a shell plugin (3rd party) doesn't have an `admin_users` option but you are right this should be in the except block.